YACS-RCOS/yacs.n
# Page for department

**Issue**
Hello Everyone,
This pull request update features on pages.
User will be redirected to a subject page Instead of using accordion for departments in Course Explorer.
All courses for a subject will be displayed into two columns.
Also, a new page is created for course detail.

*Formal Issue:*
This Pull Request still works on issue #191/ #192

**Database Changes/Migrations**
N/A

**Test Modifications**
N/A

**Test Procedure**

Show procedure to test functionality with a clear procedure

*Test New Subject Page:*
> 1. Load Spring 2020 CSV
> 2. Navigates to the course explore page by click on the blue tag "explore"
> 3. Click "CSCI" subject under the Science department
> 4. The new subject page should be displayed on http://localhost:8080/CSCI. All courses are listed "evenly" by two columns

*Test New Course Page:*
> 1. Access CSCI subject page on http://localhost:8080/CSCI.
> 2. Click "Computer Science 1 CSCI-1100" button
> 3. Click "CSCI" subject under the Science department
> 4. The new course page should be displayed on http://localhost:8080/CSCI/CSCI-1100. Detail course information should be displayed.

**Photos**
Show before and after, capture screenshot/gif of finished feature/bug
![4d63927f8af5b44567410b9f67d8599](https://user-images.githubusercontent.com/46305076/95919630-ef307180-0d7b-11eb-8dae-2e3db18a7e17.png)

![0897532793b78fae8b08d34cece6a3e](https://user-images.githubusercontent.com/46305076/95919641-f2c3f880-0d7b-11eb-8c8c-8d11c941d827.png)

**Additional Info**
This Pull Request also includes the contribution of Ike Lyons, we are using the same branch.
